lottie-ios/Classes/Private/LOTAnimationView.m

```diff
@@ -691,7 +691,11 @@ - (void)willMoveToWindow:(UIWindow *)newWindow {
 }
 
 - (void)didMoveToWindow {
+#if TARGET_OS_IOS
```

Comment:
We need to make sure this also handles macOS and tvOS right? Can you just use `TARGET_OS_VISION` and `!TARGET_OS_VISION`?